README.md

```diff
@@ -353,24 +353,22 @@ Available browser options:
 Option name               | Description
 ------------------------- | -------------
 `desiredCapabilities`     | **Required.** Used WebDriver [DesiredCapabilities](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities)
+`gridUrl`                 | Selenium grid Url. Default value is `http://localhost:4444/wd/hub`.
+`baseUrl`                 | Base service-under-test url. Default value is `http://localhost`.
```

Comment:
I don't think so. I'm tried to make variable names more similar to gemini. But in gemini it calls `rootUrl` =)